sci-misc/boinc/boinc-7.22.0.ebuild

```diff
@@ -0,0 +1,196 @@
+if [[ ${PV} == *9999 ]] ; then
+	EGIT_REPO_URI="https://github.com/BOINC/${PN}.git"
+	inherit git-r3
+else
+	SRC_URI+=" https://github.com/BOINC/boinc/archive/client_release/${MY_PV}/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+	S="${WORKDIR}/${PN}-client_release-${MY_PV}-${PV}"
+fi
+
+LICENSE="LGPL-3"
+SLOT="0"
+
+IUSE="X cuda curl_ssl_gnutls +curl_ssl_openssl opencl"
+
+REQUIRED_USE="
+	^^ ( curl_ssl_gnutls curl_ssl_openssl )
+"
+
+# libcurl must not be using an ssl backend boinc does not support.
+# If the libcurl ssl backend changes, boinc should be recompiled.
+DEPEND="
+	acct-user/boinc
+	app-misc/ca-certificates
+	cuda? (
+		x11-drivers/nvidia-drivers
+	)
+	opencl? (
+		virtual/opencl
+	)
+	dev-libs/openssl:=
+	net-misc/curl[curl_ssl_gnutls(-)=,-curl_ssl_nss(-),curl_ssl_openssl(-)=,-curl_ssl_axtls(-),-curl_ssl_cyassl(-)]
+	sys-libs/zlib
+	X? (
+		dev-libs/glib:2
+		media-libs/freeglut
+		media-libs/libjpeg-turbo:=
+		x11-libs/gtk+:3
+		x11-libs/libnotify
+		x11-libs/libX11
+		x11-libs/libXScrnSaver
+		x11-libs/libxcb:=
+		x11-libs/wxGTK:${WX_GTK_VER}[X,opengl,webkit]
+		x11-libs/xcb-util
+	)
+"
+BDEPEND="app-text/docbook-xml-dtd:4.4
+	app-text/docbook2X
+	sys-devel/gettext
+	X? ( virtual/imagemagick-tools[png,tiff] )
+"
+RDEPEND="
+	${DEPEND}
+	sys-apps/util-linux
+	!app-admin/quickswitch
+"
+
+PATCHES=(
+	# >=x11-libs/wxGTK-3.0.2.0-r3 has webview removed, bug 587462
+	"${FILESDIR}"/${PN}-7.18-fix_webview.patch
+)
+
+pkg_setup() {
+	# Bug 578750
+	if use kernel_linux; then
+		linux-info_pkg_setup
+		if ! linux_config_exists; then
+			ewarn "Can't check the linux kernel configuration."
+			ewarn "You might be missing vsyscall support."
+		elif kernel_is -ge 4 4 \
+			&& linux_chkconfig_present LEGACY_VSYSCALL_NONE; then
+			ewarn "You do not have vsyscall emulation enabled."
+			ewarn "This will prevent some boinc projects from running."
+			ewarn "Please enable vsyscall emulation:"
+			ewarn "    CONFIG_LEGACY_VSYSCALL_EMULATE=y"
+			ewarn "in /usr/src/linux/.config, to be found at"
+			ewarn "    Processor type and features --->"
+			ewarn "        vsyscall table for legacy applications (None) --->"
+			ewarn "            (X) Emulate"
+			ewarn "Alternatively, you can enable CONFIG_LEGACY_VSYSCALL_NATIVE."
+			ewarn "However, this has security implications and is not recommended."
+		fi
+	fi
+}
+
+src_prepare() {
+	default
+
+	# prevent bad changes in compile flags, bug 286701
+	sed -i -e "s:BOINC_SET_COMPILE_FLAGS::" configure.ac || die "sed failed"
+
+	eautoreconf
+}
+
+src_configure() {
+	use X && setup-wxwidgets
+
+	append-libs -L"${ESYSROOT}"/usr/$(get_libdir) -L"${ESYSROOT}"/$(get_libdir)
+
+	econf --disable-server \
+		--enable-client \
+		--enable-dynamic-client-linkage \
+		--disable-static \
+		--enable-unicode \
+		--with-ssl \
+		$(use_with X x) \
+		$(use_enable X manager) \
+		$(usex X --with-wx-config="${WX_CONFIG}" --without-wxdir)
```

Comment:
The `--libs` error comes from the econf, but I'm not able to replicate it manually. 

Downloading the source and running `./configure` with the same parameters as the ebuild seems to do less that the ebuild's `econf`, since the output its around a third of the configure phase in the ebuild. Also it doesn't output any libs error this way. `econf` seems to configure several extra things that end giving us the error at hand, but I wasn't able to find much more information in our ebuild documentation. Do you know if there is a way of debuging what `econf` calling in the background to understand where could be the problem?

Reply:
Well you can see what `econf` calls by default with **man 5 ebuild**, then search for econf. The whole string is visible there, with later EAPI additions. Although, everything should be visible in the build.log too. 

Yeah it's a weird one, thanks for taking a look! As I said, it didn't _seem_ to affect the outcome, but of course there's a chance it does.